### DISCUSSION TITLE
Please set them or make sure they are set and tested correctly in the CMake files: NetCDF_LIBRARY (ADVANCED)

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#570] [Q&A] [answered]
I can compile it successfully in ubuntu, why this happens in centos7.
![image](https://user-images.githubusercontent.com/104721358/172087509-a8cc9903-498b-4129-9efb-1cc196996967.png)

**Comment by sebwolf-de** [comment, marked as answer]
Probably the netcdf library is not properly installed. There can be various reasons:
* netcdf is installed, but in a non-standard directory.
* netcdf is compiled with another compiler than the one, you are using now
* netcdf is installed, but with a wrong version number.
